.env.template

```
@@ -181,15 +181,19 @@ SERVER_CORE_HOST=http://server-core:8080
 # SENTRY CONFIGURATION
 # ============================================================================
+# Set to 'true' to enable Sentry error tracking across all services
+SENTRY_ENABLED=false
+
 # Sentry DSN (Data Source Name) for error tracking and monitoring
-# Leave empty to disable Sentry integration
+# These are only used when SENTRY_ENABLED=true
 
 SENTRY_DSN_CORE=
 SENTRY_DSN_TEAM_ALLOCATION=
 SENTRY_DSN_SELF_TEAM_ALLOCATION=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SENTRY_DSN_SELF_TEAM_ALLOCATION key should go before the SENTRY_DSN_TEAM_ALLOCATION key

(UnorderedKey)